domains/debarchito.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "debarchito",
+           "email": "44723456+debarchito@users.noreply.github.com",
```

Comment:
Either replace the noreply email with your own email or leave it blank.

Reply:
updated!